imports/plugins/included/taxes-taxcloud/server/hooks/hooks.js

```diff
@@ -42,84 +37,85 @@ MethodHooks.after("taxes/calculate", (options) => {
         };
       }
 
-      // check if plugin is enabled and this calculation method is enabled
-      if (pkg && pkg.enabled === true && pkg.settings.taxcloud.enabled === true) {
-        if (!apiKey || !apiLoginId) {
-          Logger.warn("TaxCloud API Key is required.");
-        }
-        if (Array.isArray(cartToCalc.shipping) && cartToCalc.shipping.length > 0 && cartToCalc.items) {
-          const shippingAddress = cartToCalc.shipping[0].address;
+      if (!apiKey || !apiLoginId) {
+        Logger.warn("TaxCloud API Key is required.");
+      }
+      if (Array.isArray(cartToCalc.shipping) && cartToCalc.shipping.length > 0 && cartToCalc.items) {
+        const shippingAddress = cartToCalc.shipping[0].address;
```

Comment:
We need to not do this, but this should probably be addressed in #4089

Reply:
Any actions to take here?

Reply:
No, nothing to do for now I suppose